internal/gps/checks_test.go

```diff
@@ -0,0 +1,63 @@
+package gps
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestBasicFindIneffectualConstraints(t *testing.T) {
+	if fixtorun != "" {
+		if fix, exists := basicFixtures[fixtorun]; exists {
```

Comment:
the basic fixtures won't get you much here, as they explicitly conjoin the idea of "import" with "rule." IOW, there's no way to express an ineffectual constraint, as expressing the constraint also expresses the import.

the bimodal fixtures, however, should be able to express what you need.